wandb/plots/explain_text.py

```diff
@@ -1,6 +1,7 @@
 import wandb
 from wandb import util
-from wandb.plots.utils import test_missing, test_types, encode_labels
+from wandb.plots.utils import test_missing, test_types, encode_labels, deprecation_notice
```

Comment:
remove deprecation stuff, lets deprecate after both interfaces exist for a few releases